### PR TITLE
[VEN-2975] Chaos labs recommendations (December 12th, 2024) 

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

**Review by GitGuru7 — APPROVED** [review]
LGTM

**Review by kkirka — APPROVED** [review]
Commands and addresses LGTM

File: vips/vip-411/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        params: [BSC_USDT_MEME_IRM],
+      },
+      {
+        target: vUSDT_BSC_TRON,
+        signature: "setInterestRateModel(address)",
+        params: [BSC_USDT_DEFI_GAMEFI_TRON_IRM],
+      },
+    ],
+    meta,
+    ProposalType.REGULAR,
```
Comment (chechu) [inline vips/vip-411/bscmainnet.ts:166]: FAST_TRACK?

**Review by chechu — COMMENTED** [review]
Commands lgtm

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/415?chainId=56

**Pull request merged**